Datadog.Serverless/Datadog.Serverless.csproj

```
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

For best compatibility, I think we should target:

```suggestion
    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

(note that `TargetFrameworks` needs to be plural)

---

@lucaspimentel So just to be clear we don't need `.net6.0` in the list of target frameworks?

---

Correct. When you target `netstandard2.0`, you target a set of standard APIs that can be used in many versions of .NET, including .NET Framework 4.6.1 and above and .NET Core 2.0 (and other .NET runtimes we don't support like Mono, Xamarin, and Unity).

In theory, we could target only `netstandard2.0` and be done. However, versions of .NET Framework before 4.7.2 didn't handle that very well, so for best compatibility it is recommended to target `net461` explicitly.

From https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0:

> While NuGet considers .NET Framework 4.6.1 as supporting .NET Standard 1.5 through 2.0, there are several issues with consuming .NET Standard libraries that were built for those versions from .NET Framework 4.6.1 projects. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.